packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -0,0 +1,240 @@
+ */
+{template .active}
+	{@param spritemap: string}
+	{@param? actionItems: list<?>}
+	{@param? elementClasses: string}
+	{@param? handleSelectPageCheckboxChanged_: any}
+	{@param? handleSelectAllClicked_: any}
+	{@param? id: string}
+	{@param? selectedItems: number}
+	{@param? totalItems: number}
+
+	{let $navAttributes kind="attributes"}
+		class="management-bar management-bar-light navbar navbar-expand-md
+			{if $elementClasses}
+				{sp}{$elementClasses}
+			{/if}
+		"
+
+		{if $id}
+			id="{$id}"
+		{/if}
+	{/let}
+
+	<nav {$navAttributes}>
+		<div class="container">
+			<ul class="navbar-nav">
+				<li class="nav-item">
+					{call ClayCheckbox.render}
+						{param checked: true /}
+						{param events: ['change': $handleSelectPageCheckboxChanged_] /}
+						{param hideLabel: true /}
+						{param indeterminate: $selectedItems < $totalItems /}
+						{param label: 'Select Page' /}
+						{param ref: 'selectPageCheckbox' /}
+					{/call}
+				</li>
+				<li class="nav-item">
+					<span class="navbar-text">
+						{$selectedItems} of {$totalItems}{sp}
+						<span class="navbar-breakpoint-down-d-none">items selected</span>
+					</span>
+				</li>
+				<li class="nav-item">
+					{call ClayButton.render}
+						{param elementClasses: 'nav-btn' /}
+						{param events: ['click': $handleSelectAllClicked_] /}
+						{param label: 'Select all' /}
+						{param ref: 'selectAllButton' /}
+						{param style: 'link' /}
+					{/call}
+				</li>
+			</ul>
+			
+			{if $actionItems}
+				<ul class="navbar-nav">
+					<li class="nav-item">
+						{call ClayActionsDropdown.render}
+							{param items: $actionItems /}
+							{param spritemap: $spritemap /}
+						{/call}
+					</li>
+				</ul>
+			{/if}
+		</div>
+	</nav>
+{/template}
+
+/**
+ * This renders the component's active content.
+ */
+{template .default}
+	{@param spritemap: string}
+	{@param? elementClasses: string}
+	{@param? filterItems: list<?>}
+	{@param? handleFilterDoneButtonClick_: any}
+	{@param? handleSelectPageCheckboxChanged_: any}
+	{@param? handlePlusButtonClicked_: any}
+	{@param? handleSortingButtonClicked_: any}
+	{@param? id: string}
+	{@param? selectable: bool}
+	{@param? sortingOrder: string}
+	{@param? viewTypes: list<?>}
+
+	{let $navAttributes kind="attributes"}
+		class="management-bar management-bar-light navbar navbar-expand-md
+			{if $elementClasses}
+				{sp}{$elementClasses}
+			{/if}
+		"
+
+		{if $id}
+			id="{$id}"
+		{/if}
+	{/let}
+
+	<nav {$navAttributes}>
+		<div class="container">
+			<ul class="navbar-nav">
+				{if $selectable}
+					<li class="nav-item">
+						{call ClayCheckbox.render}
+							{param events: ['change': $handleSelectPageCheckboxChanged_] /}
+							{param hideLabel: true /}
+							{param label: 'Select Page' /}
+							{param ref: 'selectPageCheckbox' /}
+						{/call}
+					</li>
+				{/if}
+
+				{if $filterItems}
+					<li class="dropdown nav-item">
+						{call ClayDropdown.render}
```

Comment:
On the mobile this will disappear. Did you get to see that? By markup, it is necessary to add a second button...
<img width="49" alt="screen shot 2017-11-28 at 11 39 52" src="https://user-images.githubusercontent.com/13750819/33325446-0deaf5e6-d431-11e7-8777-9e9006bfa18f.png">

Reply:
This is added to the missing stuff too